TrackletAlgorithm/Constants.h

```diff
@@ -26,12 +26,12 @@ constexpr int TEBinsBits = 3;
 
 // stub digitization constants
 constexpr double kr = 0.0292969;
-constexpr double kphi = 7.8283e-06;
+constexpr double kphi = 7.71297e-06;
 constexpr double kz = 0.0585938;
 
 // tracklet digitization constants
-constexpr double krinv = 1.04377e-06;
```

Comment:
why did these change?

Reply:
I don't actually know. This changed in the emulation, so maybe @djcranshaw or @aryd know?